CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ release.
   ([#88](https://github.com/open-telemetry/semantic-conventions/pull/88))
   The conventions cover metrics that are recorded by the FaaS itself and not by
   clients invoking them.
+- Rename all JVM metrics from `process.runtime.jvm.*` to `jvm.*`
```

Comment:
Should we add the "breaking" term here as we often do? Or we don't consider these to be "breaking" since they were never stable?

Reply:
```suggestion
- BREAKING: Rename all JVM metrics from `process.runtime.jvm.*` to `jvm.*`
```